.genx/templates/react/component/component.column.defs.hbs

```diff
@@ -0,0 +1,9 @@
+import { ColDef } from '@ag-grid-community/core';
+import { getNumberFormatter, getDateFormatter } from '@genesislcap/foundation-utils';
+{{#if route.FDC3EventHandlersEnabled}}
+    import { sendEventOnChannel } from '../../../utils';
```

Comment:
nitpick - should the indentation be removed here?

Reply:
removed